_episodes/05-merging-data.md

```diff
@@ -236,7 +236,7 @@ two DataFrames based on a join key and returns a new DataFrame that contains
 DataFrames.
 
 Inner joins yield a DataFrame that contains only rows where the value being
-joins exists in BOTH tables. An example of an inner join, adapted from [this
+joined exists in BOTH tables. An example of an inner join, adapted from [this
 page](http://blog.codinghorror.com/a-visual-explanation-of-sql-joins/) is below:
```

Comment:
Let's change "this page" to "the Jeff Atwood's blogpost"
And, perhaps, a comma before `is below` ..?..